[Lab02-win] Improve skeleton TODOs and structure

teodutu opened the pull request:
Changes:
- `crc.c:`
    - Added separate files for writing and reading to and from a file,
      together with TODOs to remind students to use loops for these
      actions
    - Removed any mention of CRC from the comments on task *b*, so as to
      make it clearer that any files should be supported
    - Added the output files for the tests
- `ls.c`:
    - Fixed TODO numbers
    - Added a comment about `.` and `..` when the `-R` argument is given

teodutu committed:
[Lab02-win] Improve skeleton TODOs and structure

Changes:
- `crc.c:`
    - Added separate files for writing and reading to and from a file,
      together with TODOs to remind students to use loops for these
      actions
    - Removed any mention of CRC from the comments on task *b*, so as to
      make it clearer that any files should be supported
    - Added the output files for the tests
- `ls.c`:
    - Fixed TODO numbers
    - Added a comment about `.` and `..` when the `-R` argument is given

elena19m commented on labs/lab02/skel/win/2-crc/crc.c:
```diff
@@ -20,6 +20,11 @@
 #define CHUNKSIZE 32
 
+static void WriteCrc(int crc, HANDLE hWrite)
+{
```
Should me add some vars here?

ex (or something similar): 
```
BOOL bRet;
DWORD dwBytesWritten, dwBytesToWrite, dwBytesTotalWritten;
```

teodutu replied:
I also added similar variables to the `ReadChunk` function.

elena19m commented on labs/lab02/skel/win/2-crc/crc.c:
```diff
@@ -32,20 +37,20 @@ static void GenerateCrc(CHAR *sourceFile, CHAR *destFile)
 
-	/* TODO 1 - Write crc to destination file */
+	/* Write crc to destination file */
+	WriteCrc(crc, hWrite);
```
You should remove bytesWritten  from here.

elena19m commented on labs/lab02/skel/win/2-crc/crc.c:
```diff
@@ -60,27 +65,45 @@ static DWORD GetSize(HANDLE file)
 	return dwSize;
 }
 
+static DWORD ReadChunk(CHAR *chunk, HANDLE *hFile)
```
Why `HANDLE *hFile`? I think ` static DWORD ReadChunk(CHAR *chunk, HANDLE hFile)` would be better.

elena19m commented on labs/lab02/skel/win/3-ls/ls.c:
```diff
@@ -54,40 +53,40 @@ static void ListFiles(TCHAR *path, BOOL bRecursive, BOOL bAll)
-	 * TODO 1 -  Find the first file in the directory
+	 * TODO 1 - Find the first file in the directory
 	 * Use extendePath as argument
 	 */
```
Fix typo: extende**d**Path instead of extendePath.

teodutu committed:
[Lab02-win] Apply feedback for skeleton

Changes:
- `crc.c`:
  - Added variables to the `ReadChunk` and `WriteCrc` functions
  - Removed unnecessary variables from `GenerateCrc`
  - Fixed the second parameter for the `ReadChunk` function
- `ls.c`:
  - Fixed a typo in a TODO commend
